### ISSUE TITLE
Intel logo on Homepage.

### ISSUE THREAD

**Issue by bajiraoayush** [AIDeveloperCommunity/aidevelopercommunity.github.io#40]
The Intel logo on the homepage would look better with some supporting text filling the black emptiness.

**Comment by crespoter** [comment]
@bajiraoayush 
That section was meant for  all the sponsors. Currently since we only have one the other spots are vacant
@SriHarshaGajavalli  what do you think?

**Comment by soham1705** [comment]
Should we have some text saying 'Sponsors' or something like that?

**Comment by crespoter** [comment]
@soham1705  that is a good suggestion

**Comment by SriHarshaGajavalli** [comment]
@soham1705: Well said. We definitely need something there. How about "Supporters" rather than "Sponsors"?

And Intel logo needs to be changed, I will update the original one soo.

**Comment by soham1705** [comment]
'Supporters' seems fine, though 'Sponsors' sounds more official.

**Comment by SriHarshaGajavalli** [comment]
True. Just thinking "Supportes" or exact synonyms covers more (kind of superset for sponsors)

**Comment by soham1705** [comment]
Yeah, I agree with that.

**Comment by crespoter** [comment]
But sponsors sound more apt for the context
